packages/web/cpp/renderer.cpp

```diff
@@ -177,6 +208,14 @@ class __attribute__((visibility("default"))) Renderer
         animation->picture()->scale(scale);
         animation->picture()->translate(shiftX, shiftY);
 
+        if (background)
+        {
+
+            background->reset();
+            background->appendRect(shiftX, shiftY, psize[0], psize[1]);
+            background->scale(scale);
```

Comment:
@hermet when the background is not scaled like the picture, it doesn't align properly on the canvas, as shown in the screenshot. Is this acceptable, or is there a more effective solution?

![image (1)](https://github.com/LottieFiles/dotlottie-web/assets/39750790/580a708d-1e8f-4f8b-8f09-4cd215e91f2d)